package lotte.newdevps.exception.place;

import lotte.newdevps.exception.http.NotFoundException;

public class PlaceNotFoundException extends NotFoundException {
}
